src/browser.js

```diff
@@ -213,7 +213,8 @@ function load() {
 	}
 
 	// If debug isn't set in LS, and we're in Electron, try to load $DEBUG
-	if (!r && typeof process !== 'undefined' && 'env' in process) {
+	// process.env is undefined in sometimes, and it will throw a exception
+	if (!r && typeof process !== 'undefined' && 'env' in process && process.env !== undefined) {
```

Comment:
The second two checks can be simplified as just `process.env`.

Reply:
Ok, I changed it